test/headful.spec.js

```diff
@@ -76,6 +76,14 @@ module.exports.addTests = function({testRunner, expect, defaultBrowserOptions})
       expect(await page.evaluate(() => 2 * 3)).toBe(6);
       await browserWithExtension.close();
     });
+    it('target.title() should return extension title', async({}) => {
+      const browserWithExtension = await puppeteer.launch(extensionOptions);
+      const backgroundPageTarget = await waitForBackgroundPageTarget(browserWithExtension);
+      const backgroundPageTitle = backgroundPageTarget.title();
+      expect(backgroundPageTitle === 'Simple extension' ||
+        backgroundPageTitle === 'CryptoTokenExtension').toBeTruthy();
```

Comment:
I’d rather not have the crypto token leak into our test. How about just getting a page, setting the title, then doing page.target.title?

Reply:
> How about just getting a page, setting the title, then doing page.target.title?

No problem, but is the `target.spec.js` test sufficient without a test here then?